src/bridge.ts

```diff
@@ -31,6 +31,8 @@ interface ProcessRef {
 
 interface RunOptions {
   pipeOutput?: boolean
+  env?: NodeJS.ProcessEnv
+  extendEnv?: boolean
```

Comment:
In what case would we set `extendEnv` to `true`? Or is this for backwards compatibility?

Reply:
The default value is `true` atm. I was hardcoding this at first, but then wondered if `runInBackground` might be used in other scenarios in the future. Do you think I should hardcode this? 
And because `run` does the same, besides the background part I also added the options there.